tools/hdf_converter/hdf_converter.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+import optparse
+import sys
+import warnings
+
+import pandas as pd
+
+
+warnings.simplefilter('ignore')
+
+parser = optparse.OptionParser()
```

Comment:
optparse is [deprecated](https://docs.python.org/3/library/optparse.html), in future work please look into using [argparse](https://docs.python.org/3/library/argparse.html#module-argparse)